docs/tutorial/category_theory/category_theory_intro.md

```diff
@@ -0,0 +1,181 @@
+# An introduction to category theory in Lean
+
+This is an introduction to the basic usage of category theory (in the mathematical sense) in Lean. We cover how the basic theory of categories, functors and natural transformations is set up in Lean. Most of the below is not hard to read off from the files `category_theory/category.lean`, `category_theory/functor.lean` and `category_theory/natural_transformation.lean`, but it is a bit painful having to constantly have all three files open at once!
+
+
+First a word of warning. In `mathlib`, in the `/src` directory, there is a subdirectory called `category`. This is *not* where categories, in the sense of mathematics, are defined; it's for use by computer scientists. The directory we will be concerned with here is the `category_theory` subdirectory.
+
+The theory was largely developed by Scott Morrison, with the help of (who else to mention? **TODO**)
+
```

Comment:
```suggestion
```
I suggest we just remove. I might add a note somewhere about the role of automation in the category theory library.